VideoLocker/res/layout/fragment_add_response.xml

```diff
@@ -21,6 +21,8 @@
             android:paddingRight="@dimen/edx_margin"
             android:paddingTop="@dimen/widget_margin">
 
+            <include layout="@layout/discussion_user_profile_row" />
```

Comment:
Also, why are the screen for adding a new responses and comments being modified to show the user profile, but not the one for adding new posts?